src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/CustomConfigTests.kt

```diff
@@ -82,6 +83,12 @@ abstract class CustomConfigTests
     {
         spark.Spark.stop()
 
+        while (isSparkInstanceRunning())
```

Comment:
Moved to doing this before database delete to avoid the weird error messages we were getting when requests were fired off by a page, then the test cleaned up and db deleted before they were served (all those `table does not exist` errors in the test logs previously).